seumich/static/seumich/sort_table.js

```diff
@@ -1,3 +1,19 @@
 $(document).ready(function() {
-    $(".table").tablesorter();
+    $(".table").tablesorter({
+        theme: "bootstrap",
+        headerTemplate: "{content} {icon}",
+        widgets: [ "uitheme", "filter", "columns", "zebra" ]
+    })
+    .tablesorterPager({
+        // target the pager markup - see the HTML block below
+        container: $(".ts-pager"),
+        // target the pager page select dropdown - choose a page
+        cssGoto  : ".pagenum",
+        // remove rows from the table to speed up the sort of large tables.
+        // setting this to false, only hides the non-visible rows; needed if you plan to add/remove rows with the pager enabled.
+        removeRows: false,
+        // output string - default is '{page}/{totalPages}';
+        // possible variables: {page}, {totalPages}, {filteredPages}, {startRow}, {endRow}, {filteredRows} and {totalRows}
+        output: "{startRow} - {endRow} / {filteredRows} ({totalRows})"
```

Comment:
need to make this output line more readable?

As included in the Slack screenshot, this line is 1-7/7(7), which might be hard for user to understand.

Reply:
Yeah, we can improve this output line. I'll find a quick slot to do a demo today and improve this.